catalog.html

```diff
@@ -51,7 +56,51 @@ <h1>Моноподы для селфи</h1>
 
       <section>
         <h3>Фильтр:</h3>
-        <a href="blank.html">Показать</a>
+        <form action="blank.html" method="get">
+          <fieldset>
+            <legend>Цвета</legend>
+            <ul>
+              <li>
+                <input type="checkbox" name="color-choice" id="filter-color-black" checked>
+                <label for="filter-color-black">Чёрный</label>
+              </li>
+              <li>
+                <input type="checkbox" name="color-choice" id="filter-color-white">
+                <label for="filter-color-white">Белый</label>
+              </li>
+              <li>
+                <input type="checkbox" name="color-choice" id="filter-color-blue" checked>
+                <label for="filter-color-blue">Синий</label>
+              </li>
+              <li>
+                <input type="checkbox" name="color-choice" id="filter-color-red" checked>
+                <label for="filter-color-red">Красный</label>
+              </li>
+              <li>
+                <input type="checkbox" name="color-choice" id="filter-color-pink">
+                <label for="filter-color-pink">Розовый</label>
+              </li>
+            </ul>
+            </fieldset>
+        </form>
+
+        <form action="blank.html" method="get">
+          <fieldset>
+            <legend>Bluetooth</legend>
+            <ul>
+              <li>
+                <input type="radio" name="option-choice" value="have-bluetooth" id="filter-have" checked>
+                <label for="filter-have">Есть</label>
+              </li>
+              <li>
+                <input type="radio" name="option-choice" value="dont-have-bluetooth" id="filter-dont-have">
+                <label for="filter-dont-have">Нет</label>
+              </li>
+            </ul>
+          </fieldset>
+        </form>
+
+        <button class="button" type="button">Показать</button>
```

Comment:
```suggestion
        <button class="button" type="submit">Показать</button>
```

Эта кнопка часть формы, а значит размечаем её в форме.

Reply:
Исправил